Calculator.Maicosoft/CalculatorLibrary/Calculate.cs

```diff
@@ -0,0 +1,77 @@
+﻿using Newtonsoft.Json;
+
+namespace CalculatorLibrary;
+
+public class Calculate
+{
+    JsonWriter _writer;
+    public Calculate()
+    {
+        StreamWriter logFile = File.CreateText("Calculate.json");
+        logFile.AutoFlush = true;
+        _writer = new JsonTextWriter(logFile)
+        {
+            Formatting = Formatting.Indented
+        };
+        _writer.WriteStartObject();
+        _writer.WritePropertyName("Operations");
+        _writer.WriteStartArray();
+    }
+    public string DoMath(double num1, double num2, string operation)
+    {
+        double result =0;
+        string output = string.Empty;
+        string op = operation;
+
+        while (!new[] {"a", "s", "m", "d"}.Contains(op))
+        {
+            Console.WriteLine("Not a valid input, Please enter an operation: ");
+            op = Console.ReadLine();
+        }
+
+        _writer.WriteStartObject();
```

Comment:
If I use the calculator twice, I get this error : 
![image](https://github.com/TheCSharpAcademy/CodeReviews.Console.Calculator/assets/108711476/8b152326-4837-4236-9ea6-9cfc42ae22d1)

Reply:
Thanks for reviewing my code, it's really a great platform to learn C# by actually doing!

I made some changes, I think this version must do the trick :-)